src/main/fc/rc.c

```diff
@@ -75,7 +72,7 @@ static float rcCommandDivider = 500.0f;
 static float rcCommandYawDivider = 500.0f;
 
 FAST_DATA_ZERO_INIT uint8_t interpolationChannels;
-static FAST_DATA_ZERO_INIT uint32_t rcFrameNumber;
+static FAST_DATA_ZERO_INIT bool newRxDataForFF;
```

Comment:
Correct camel casing here would be `newRxDataForFf` (consistent with `getShouldUpdateFf()`).